### ISSUE TITLE
Margins xs,sm,md ...

### ISSUE THREAD

**Issue by Randerspl** [Spiderpig86/Cirrus#68]
**Is your feature request related to a problem? Please describe.**
Hello i want to ask about margins.
Cirrus declares margin classes like m-1 or mb-1 which is very helpfull.
I know that cirrus can do something like flex-column on small device and row on medium (i saw this in docs with version 0.6.2).
Im asking if margins can have similar functionality like on small devices add margin bottom (because of column layout) and on md or more don't (because of row layout).

**Describe the solution you'd like**
Something like:
```
<div class="u-flex u-flex-column u-flex-row-md">
  <button class="mb-1 mb-0-md"> Button1 </button>
  <button class="mb-1 mb-0-md"> Button2 </button>
</div>
```

On small screen buttons have margins because they are in column and needs some space between. On row style they don't need one on bottom.

Maybe i don't know something and exists solution for this one in Cirrus?

**Comment by Spiderpig86** [comment]
Hi! This is not something Cirrus supports currently, but I can work on adding it for the next version. I intentionally left out the viewport sizing for all possible util classes to keep the builds small and will add them based on demand.

**Comment by Randerspl** [comment]
In my opinion responsive margins will be useful (maybe paddings too?), especially when responsive flex layout already exists

**Comment by Spiderpig86** [comment]
Resolved as of 0.6.3 release. Closing.
